### DISCUSSION TITLE
Inline content remove is-draggable

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#456] [Q&A]
Hello! 
How to remove .is-draggable class from the fancybox__viewport block for  inline content?

**Comment by fancyapps** [comment]
Hi,

Please, upgrade to the latest (v5.0.11), it should not be applied unless you have multiple slides in gallery.

**Reply by oshumelka** [reply]
Hi! I've just did it, but fancybox__viewport  still has .is-draggable class
Should I add additional options?

**Reply by fancyapps** [reply]
No, it is not added https://jsfiddle.net/mjz582wg/

**Reply by oshumelka** [reply]
Thanks for your answer! It's something strange, but I still see .is-draggable class after update. I hope I manage with this.
My another question is:
How to select text within popup with a mouse?

**Reply by fancyapps** [reply]
1) It would be great if you could make a demo to test this;
2) Add `data-selectable` attribute to the whole content or only the parts you wish to make selectable. You can combine that with `dragToClose: false`, demo - https://jsfiddle.net/yuvfz98j/
